website/content/en/docs/deployment/prerequisites.md

```diff
@@ -81,5 +81,28 @@ The command above installs the following tools:
 - [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) - An infrastructure as code tool that lets you develop cloud and on-prem resources.
 - [helm](https://helm.sh/docs/intro/install/) - A package manager for Kubernetes
 
+## Configure AWS CLI
+
+To access AWS services with the AWS CLI, you need an AWS account, IAM credentials, and an IAM access key pair. When running AWS CLI commands, the AWS CLI needs to have access to those AWS credentials. 
+
+Note that your account must have **administrative privileges**. 
+
+Run the following command to configure AWS CLI:
+
+> Warning [Cloud9](https://docs.aws.amazon.com/cloud9/latest/user-guide/security-iam.html) users: To prevent Cloud9 from overwriting the IAM roles with temporary credentials, disable the temporary credential management within Cloud9 before running the aws configure command below. In `AWSCloud9 Console`, go to `AWS Settings` then disable `AWS Managed temporary credentials`.
```

Comment:
> To prevent Cloud9 from overwriting the IAM roles with temporary credentials, disable the temporary credential management within Cloud9 before running the aws configure command below. In `AWSCloud9 Console`, go to `AWS Settings` then disable `AWS Managed temporary credentials`.

^^ This does did not work.

Following worked
 
When configuring credentials on Cloud9 we recommend configuring a [profile with the credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-profiles). Click `Cancel` when it opens a pop up "could not update credentials", and click `Permanently disable` for the "unable to update credentials" pop up